common/source/docs/common-frsky-repurposed.rst

```diff
@@ -0,0 +1,95 @@
+.. _common-frsky-repurposed:
+
+FrSky Telemetry Using Repurposed Messages
+=========================================
+
+For information on how to connect the FrSky equipment together, please go :ref:`here <common-frsky-equipment>`.
+
+For information on how to configure ArduPilot for FrSky telemetry, please go :ref:`here <common-frsky-configMP>`. 
+
+Once your equipment is connected and ArduPilot is configured, follow the instructions below to get standard FrSky telemetry displayed on your RC transmitter.
+
+Transmitter set-up
+------------------
+
+Please refer to the `OpenTX manual <https://docs.google.com/document/d/1qlh09LzxtpPt7j_aqG8yiOu2yoYMzP9XA-PJA81rDJQ/edit#heading=h.36trni4byo5x>`__
```

Comment:
Again, pointing to an external google docs. Perhaps pointing to the gitbook page ?